### PR TITLE
[Product Model] creando modelo de productos

### PR DESCRIPTION
**Pull request by nestornovella**
descript: 
created products model...

### CODE REVIEW

**Review by Ariel-Alegre — APPROVED** [review]

File: api/database/models/products.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+
+// module.exports = (sequelizeInstance)=>{
+//     sequelizeInstance.define("Products",{
+//         id:{
+//             type:DataTypes.STRING,
+//             primaryKey: true,
+//             allowNull: false,
+//         },
+//         name:{
```
Comment (miguelbel00) [inline api/database/models/products.js:62]: unnecessary comments

File: api/database/models/products.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            type:DataTypes.STRING,
+            allowNull: false
+        },
+        description:{
+            type:DataTypes.TEXT,
+            allowNull: false
+        },
+        image:{
+            type:DataTypes.ARRAY(DataTypes.STRING),
+            allowNull: false
```
Comment (miguelbel00) [inline api/database/models/products.js:34]: allowNull should be true

File: api/database/models/products.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  module.exports = (sequelize, DataTypes) => {
+    class Products extends Model {
+      /**
+       * Helper method for defining associations.
+       * This method is not a part of Sequelize lifecycle.
+       * The `models/index` file will call this method automatically.
+       */
+      static associate(models) {
+        // define association here
+        /* Test.hasMany(models.User,{
```
Comment (miguelbel00) [inline api/database/models/products.js:13]: delete this comments if you don't use

**Review by miguelbel00 — CHANGES_REQUESTED** [review]
Folder src it's unnecessary  and what its the function of the idCreator? why no uuid?

**Comment by miguelbel00** [comment]
don't have the evidence of the model created in database

**Comment by nestornovella** [comment]
eliminado porque habia errores en la creacion de modelos

**Pull request closed without merging**